HelpSource/Guides/standalones.schelp

```diff
@@ -1,212 +1,349 @@
+::
 
-## strong::Customize Help::
+subsection:: How this guide works
 
-	Edit Resources/HelpSource/Help.schelp to inform users about YourApp.
+This guide is a set of scripts for creating a SuperCollider-based Standalone app in a few basic steps:
 
-## strong:: Add Resources, Extensions, etc.::
+numberedlist::
+## copy the open SuperCollider.app or SC3-derived standalone.
 
-	LIST::
-	## Put any soundfiles you intend to use in /Contents/Resources/sounds
-	## Put any extra UGen plugin libraries your app requires in (e.g. Kloink.scx) in Resources/plugins
-	## Put any classes for YourApp in Resources/SCClassLibrary
-	::
+## fix the copy's info files so it uses its own userAppSupportDir
 
-## strong::Startup File::
+## test that Platform.userAppSupportDir is independent in the new app!
 
-	In Resources/SCClassLibrary/, modify the file modifyStartup.sc to do anything your app needs at startup, such as boot the server, instantiate classes and run code... See the section link::#Custom Startup:: below.
+... Then the adaptation to the specific project begins: ...
 
-## strong::Test Your Application::
+## transfer active quarks and extensions to the new app
 
-	Start YourApp and test that it works. You can recompile, edit scripts, add classes, as in SuperCollider itself.
+## copy the project folder used to the new app
 
-## When your app works as desired - strong::Share and Enjoy!::
+## make an internal startupFile for it
 
+## test and adapt until it does what you want
 ::
 
-section:: Creating from Scratch
-
-subsection:: Basic Setup and Building
-
-NUMBEREDLIST::
+subsection:: What to prepare
 
-## strong::Obtain SuperCollider Source Code::
+list::
+## have your system ready with all desired quarks installed,
+and a working startup file. (can be a very simple test setup)
 
-	The easiest way to do so is to download the latest source release from SourceForge:
-	http://supercollider.sourceforge.net/downloads/
+## read each section, then try evaluating it at once;
+ in case of problems, evaluate line by line,
+ and tell me where which problems happen
 
-## strong::Open XCode Project::
-
-	Open the XCode project located at strong::platform/mac/Language.xcodeproj:: within the SuperCollider sources.
-
-## strong::Name Your Application::
-
-	Rename the "Standalone Template" target by control-clicking it and choosing "Rename". This target name is used for both the .app name and for the name presented in the menu bar.
-
-image::standalones_rename.png::
+## when you get to the wakeup kiss, share and enjoy!
+::
 
-## strong::Add Resources::
+subsection:: Naming, copying, and fixing up the new app
 
-	Customize the "Standalone Resources" directory, which is located at the root of the SuperCollider source directory. Any files you put in this directory will override the files originating from the SuperCollider source at the same destination.
+// TO PREPARE, CHECK YOUR CURRENT config in Preferences->Interpreter:
```

Comment:
I think you intended this to go in the code:: block below; when rendered as-is this does not look well-formatted.